objects/src/lib.rs

```diff
@@ -10,20 +10,20 @@ use assembly::{
 };
 use crypto::{
     hash::rpo::{Rpo256 as Hasher, RpoDigest as Digest},
-    merkle::{MerkleError, MerkleStore, Mmr, TieredSmt},
+    merkle::{MerkleError, Mmr, TieredSmt},
     utils::{
-        collections::{BTreeMap, Vec},
+        collections::Vec,
         string::{String, ToString},
     },
     Felt, StarkField, Word, WORD_SIZE, ZERO,
 };
 use vm_core::code_blocks::CodeBlock;
-use vm_processor::{AdviceInputs, StackOutputs};
+use vm_processor::AdviceInputs;
 
-mod accounts;
+pub mod accounts;
 pub use accounts::{
     validate_account_seed, Account, AccountCode, AccountDelta, AccountId, AccountStorage,
-    AccountStub, AccountType, AccountVault, StorageItem, DEFAULT_ACCOUNT_CODE,
+    AccountStub, AccountType, AccountVault, StorageItem,
 };
 
 mod advice;
```

Comment:
One thing I realized is that we should probably move `advice` module and all related to code to `miden-tx` crate as well. But we can do it in a different PR.

Reply:
https://github.com/0xPolygonMiden/miden-base/issues/255